packages/react/src/products/hooks/types/useProductListing.ts

```diff
@@ -15,6 +15,7 @@ export interface UseProductListingCommonOptions {
   useCache?: boolean;
   enableAutoFetch?: boolean;
   setProductsListHash?: boolean;
+  isACustomListingPage?: boolean;
```

Comment:
You can name it like this to be more consistent:

```suggestion
  isCustomListingPage?: boolean;
```